Footer dev message not shown when exist theme app extension (#1533)
Shopify/cli: .changeset/clean-donuts-brush.md

---
'@shopify/app': patch
---

> Footer dev message not shown when exist theme app extension. Plain app urls is shown instead"

Reviewer: ```suggestion
Fix URLs output for dev with theme app extensions
```
